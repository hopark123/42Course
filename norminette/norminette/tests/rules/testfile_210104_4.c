int	((*g_conv[13])(t_syntax syntax, t_buffer *buffer, va_list va));
