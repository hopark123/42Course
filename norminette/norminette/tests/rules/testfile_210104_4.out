[36mtestfile_210104_4.c[0m - [32mIsVarDeclaration[0m In "GlobalScope" from "None" line 1":
		<INT> <TAB> <LPARENTHESIS> <LPARENTHESIS> <MULT> <IDENTIFIER=g_conv> <LBRACKET> <CONSTANT=13> <RBRACKET> <RPARENTHESIS> <LPARENTHESIS> <IDENTIFIER=t_syntax> <SPACE> <IDENTIFIER=syntax> <COMMA> <SPACE> <IDENTIFIER=t_buffer> <SPACE> <MULT> <IDENTIFIER=buffer> <COMMA> <SPACE> <IDENTIFIER=va_list> <SPACE> <IDENTIFIER=va> <RPARENTHESIS> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
testfile_210104_4.c: OK!
